sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 # Release History
 
-## 1.5.0-beta.1 (Unreleased)
+## 1.5.0 (Unreleased)
 
 ### Features Added
+- Introduction of new scopes for token generation.
+    - `chat.join` (Access to Chat APIs but without the authorization to create, delete or update chat threads)
```

Comment:
```suggestion
    - `CHAT_JOIN` (Access to Chat APIs but without the authorization to create, delete or update chat threads)
```
let's use uppercase for the scopes as they are defined in the CommunicationTokenScope, same comment for the readme

Reply:
Will do this.